tsconfig.build.json

```diff
@@ -0,0 +1,7 @@
+{
+  "files": [],
+  "references": [
+    { "path": "./tsconfig.esm.json" },
+    { "path": "./tsconfig.test.json" }
+  ]
+}
```

Comment:
Should this be the default `tsconfig.json`?

Reply:
I left every project in the default tsconfig.json for the IDE and removed the benchmark from the tsconfig.build.json

Reply:
We agree. That's what is implemented here: `tsconfig.json` references every sub project including `@itty-aws/benchmark` for the IDE.

But this is `tsconfig.build.json` that we are talking about, and I removed `benchmark/tsconfig.json` from it since the benchmark sub-project is not to be build through the main `pnpm build` command. In this subproject, the CDK takes care of building the lambda functions automatically with `esbuild` when deploying through `pnpm bench:deploy`.